packages/shipjs-lib/src/lib/git/getRemoteBranches.js

```diff
@@ -11,13 +11,19 @@ $ git branch -r
 */
 
 export default function getRemoteBranches(dir = '.') {
-  const remote = silentExec('git remote', { dir }).toString().trim();
+  const origins = silentExec('git remote', { dir })
+    .toString()
+    .trim()
+    .split('\n');
   return silentExec('git branch -r', { dir })
     .toString()
     .trim()
     .split('\n')
     .map((line) => line.trim())
-    .filter(Boolean)
     .filter((line) => !line.includes(' -> '))
-    .map((line) => line.slice(remote.length + 1));
+    .map((line) => {
+      const origin = origins.find((_origin) => line.startsWith(`${_origin}/`));
+      return origin ? line.slice(origin.length + 1) : null;
```

Comment:
origin can't really be undefined, maybe it makes more sense for an error (can't read slice of undefined) than dealing with `null` as a possible output. What do you think?

Reply:
Yeah, thinking about it again, it makes sense origins and branch names are synchronised in a way that `origin` variable must be truthy.